app/models/project.rb

```diff
@@ -929,6 +932,19 @@ def archive!
     update_attribute :status, STATUS_ARCHIVED
   end
 
+  def required_storage
+    Rails.cache.fetch("project##{id}/project_storage", expires_in: 1.hour) do
+      storage, = count_for(:required_project_storage)
```

Comment:
I really thought to see a typo here. But I believe you are intentionally putting that `,` there...

I did not yet find the advantage over `count_for(...).first`, which is what I would propose ^^